ui/src/main/js/application/TopNavBar.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+import { NavLink } from 'react-router-dom';
+import BlackDuckLogo from '/src/main/resources/BlackDuckLogo.png';
```

Comment:
Should we follow ESLint styling rules to make paths relative?

Reply:
For Alert we have made changes to the build to make the imports from the root project directory so that you can actually find the item being imported.  With relative paths and the tools we had when we started Alert refactoring and moving classes did not always adjust the import paths correctly.  Determining which part of the relative path caused the issue was difficult at times.

Reply:
Yeah I wanted to use relative paths but referring to Paulo's comment as an answer.  For now I'll leave as is.  Perhaps in a future release we could look to update our absolute paths to relative paths.